encoding/protobq/integration_test.go

```diff
@@ -95,6 +95,17 @@ func Test_Integration_PublicDataSets(t *testing.T) {
 			Limit:     10,
 			Message:   &publicv1.WhosOnFirstGeoJson{},
 		},
+
```

Comment:
Commit message says this doesn't pass. Is that correct?

Reply:
Yes, but it will in the next upcoming PR. :smile: